src/Lean/Meta/Tactic/AC/Main.lean

```diff
@@ -0,0 +1,178 @@
+/-
+Copyright (c) 2022 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dany Fabian
+-/
+import Init.Data.AC
+import Lean.Meta.AppBuilder
+import Lean.Elab.Tactic.Basic
+import Lean.Elab.Tactic.Rewrite
```

Comment:
I don't think we should import Lean.Elab from Lean.Meta?

Reply:
Thanks for catching this. We should split this file into `Meta` and `Elab` (frontend) parts.